types/fingerprintjs__fingerprintjs/index.d.ts

```diff
@@ -0,0 +1,122 @@
+// Type definitions for @fingerprintjs/fingerprintjs 2.1
+// Project: https://github.com/fingerprintjs/fingerprintjs
+// Definitions by: Curt Mullin <https://github.com/curtstate>
+//                 BendingBender <https://github.com/BendingBender>
+//                 Luke Bennett <https://github.com/LukeeeeBennett>
```

Comment:
```suggestion
// Definitions by: Luke Bennett <https://github.com/LukeeeeBennett>
```
Unless you name them here in PR to get explicit OK, I think that would be better to 
not include handles without their visible ACK